commands/pr-id.sh

```diff
@@ -18,7 +21,16 @@ pr-id() {
         }
       }
     }
-  ' | jq -r '.data.repository.pullRequests.edges[0].node[]' 
+  ' | jq -r '(.data.repository.pullRequests.edges[0].node // {})[]')"
+
+  (( "$?" )) && return 1
+
+  if [[ -z "$result" ]]; then
+    echo 'The pull request was not found.'
```

Comment:
I want to use stderr

```suggestion
    echo "The pull request was not found." 1>&2;
```